recipes/kakoune/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "Kakoune" %}
+{% set version = "2024.05.18" %}
+{% set author = "mawww" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ author }}/{{ name|lower }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 50f2920db8ab8f71556b73dfd6d53fb924f67d1c60b9882050cfecfaa3aed31f
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ stdlib("c") }}
+    - llvm  # [osx]
```

Comment:
This should come in via the compiler or does it explicitly use LLVM features?
```suggestion
```

Reply:
> This should come in via the compiler 
As you say it does.

> does it explicitly use LLVM features?
Stuff coming with the compiler seems enough for the package to work.